NDLANO/editorial-frontend
# Added H3 support, translated toolbar icon titles

Resolves https://github.com/NDLANO/Issues/issues/1098

Test Procedure:
- Check that `H3` tag appears in Editor toolbar
- Check that using `H3` transforms the text into the correct formatting when using the toolbar
- Check that `H3`s are saved and re-appearing after reloading of article
- Check that correct title texts appears on all toolbar icons
